### PR TITLE
WT-13625 Add python reproducer for WT-12602

### PR DESCRIPTION
**Pull request by jiechenbo**
Add in WT-12602 reproducer that was accidentally left out in the pull request.

### CODE REVIEW

File: test/suite/test_bug034.py [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+
+        # Reconcile all data onto the disk.
+        self.session.checkpoint()
+
+        # Peform dirty eviction to trigger reconcilliation.
+        self.session.begin_transaction()
+        for i in range(1, self.nrows):
+            cursor[str(i)] = value1
+        self.session.commit_transaction('commit_timestamp=' + self.timestamp_str(11))
+        self.evict_cursor(uri, self.nrows)
```
Comment (etienneptl) [inline test/suite/test_bug034.py:168]:
```suggestion
        self.evict_cursor(uri, self.nrows)

```

File: test/suite/test_bug034.py [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+        # to the HS.
+        self.session.begin_transaction()
+        for i in range(1, self.nrows):
+            cursor[str(i)] = value1
+        self.session.commit_transaction()
+
+        # Reconcile all data onto the disk.
+        self.session.checkpoint()
+
+        # Peform dirty eviction to trigger reconcilliation.
```
Comment (etienneptl) [inline test/suite/test_bug034.py:107]:
```suggestion
        # Perform dirty eviction to trigger reconciliation.
```

File: test/suite/test_bug034.py [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+        # to the HS.
+        self.session.begin_transaction()
+        for i in range(1, self.nrows):
+            cursor[str(i)] = value1
+        self.session.commit_transaction('commit_timestamp=' + self.timestamp_str(10))
+
+        # Reconcile all data onto the disk.
+        self.session.checkpoint()
+
+        # Peform dirty eviction to trigger reconcilliation.
```
Comment (etienneptl) [inline test/suite/test_bug034.py:163]:
```suggestion
        # Perform dirty eviction to trigger reconciliation.
```

Comment (jiechenbo) [reply test/suite/test_bug034.py]: [f8c40d6](https://github.com/wiredtiger/wiredtiger/pull/11105/commits/f8c40d6b2272baaf61390448222fbda5e124b494)

File: test/suite/test_bug034.py (hunk 2 above)
Comment (jiechenbo) [reply test/suite/test_bug034.py]: [f8c40d6](https://github.com/wiredtiger/wiredtiger/pull/11105/commits/f8c40d6b2272baaf61390448222fbda5e124b494)

File: test/suite/test_bug034.py (hunk 1 above)
Comment (jiechenbo) [reply test/suite/test_bug034.py]: [f8c40d6](https://github.com/wiredtiger/wiredtiger/pull/11105/commits/f8c40d6b2272baaf61390448222fbda5e124b494)

**Review by etienneptl — APPROVED** [review]
LGTM, thank you for adding the missing repro!

**Pull request merged**